public/index.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
```

Comment:
Didn't see where we set the order for the menu bar, but we should swap reference and tutorials:
Install | Tutorials | Reference | GitHub | Blog